### ISSUE TITLE
Missing navigation on https://www.w3.org/WAI/WCAG21/Understanding/reflow

### ISSUE THREAD

**Issue by juleskuehn** [w3c/wcag#375]
Unlike other pages falling under "Understanding WCAG 2.1", the linked page lacks navigation elements to access its adjacent or parent pages.

https://www.w3.org/WAI/WCAG21/Understanding/reflow

**Comment by awkawk** [comment]
closing as fixed.
